### DISCUSSION TITLE
container doesn’t support @media (min-width: 768px): or any other variants

### DISCUSSION THREAD

**Discussion by rossthedevigner** [ben-rogerson/twin.macro#512] [Q&A] [answered]
I'm trying to use the example found in the docs to make my element behave like a container at certain sizes: https://tailwindcss.com/docs/container#responsive-variants

```
<!-- Full-width fluid until the `md` breakpoint, then lock to container -->
<div class="md:container md:mx-auto">
  <!-- ... -->
</div>
```

However, I get this error: 
```
container doesn’t support @media (min-width: 768px): or any other variants

/node_modules/twin.macro/macro.js:6677:13
/node_modules/twin.macro/macro.js:5965:18)
```

I'm on version 2.2.7 of tailwind and 2.6.2 of twin.macro. You can see it working here in the first `div` element https://play.tailwindcss.com/m8IszGZkJR

**Comment by ben-rogerson** [comment]
Looks like an easy fix, I'll just enable variants on the container class 👍

**Comment by ben-rogerson** [comment, marked as answer]
I've added [a PR fix](https://github.com/ben-rogerson/twin.macro/pull/518) for including in the next version of twin.

**Reply by rossthedevigner** [reply]
I totally missed this addition! Thank you for jumping on it so fast - really loving this project 💯
